site2/docs/reference-pulsar-admin.md

```diff
@@ -2267,6 +2252,30 @@ Options
 |`-l`, `--limit`|Size limit (eg: 10M, 16G)", required = true)||
 |`-p`, `--policy`|Retention policy to enforce when the limit is reached. The valid options are: `producer_request_hold`, `producer_exception` or `consumer_backlog_eviction`|
 
+### `remove-deduplication`
+Remove a deduplication policy from a topic.
+
+Usage
+```bash
+$ pulsar-admin topics remove-deduplication tenant/namespace/topic
+```
+
+### `get-backlog-quotas`
+Get the backlog quota policies for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics get-backlog-quotas tenant/namespace/topic
+```
+
+### `set-backlog-quota`
+Set a backlog quota policy for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics set-backlog-quota tenant/namespace/topic
```

Comment:
should document the parameters defined here: https://github.com/apache/pulsar/blob/master/pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdTopics.java#L984-L989